src/frontend/src/env/tokens.btc.env.ts

```diff
@@ -30,7 +30,22 @@ export const BTC_TESTNET_TOKEN: Token = {
 	standard: 'bitcoin',
 	category: 'default',
 	name: 'Bitcoin (Testnet)',
-	symbol: 'BTC (Testnet)',
+	symbol: BTC_TESTNET_SYMBOL,
+	decimals: BTC_DECIMALS,
+	icon: bitcoinTestnet
```

Comment:
Regtest and testnet do not have different logo?

Reply:
We don't have a logo for regtest yet and we thought we leave the same as bitcoin testnet.

Regtest is only available for local development, no user will see it.

Reply:
Does it needs to be the same logo then?

If you don't specify a logo then it default to a color generate based on the name of the token.

Reply:
Maybe we should remove the logo and use the default. Wdyt @DenysKarmazynDFINITY ?

Reply:
Totally, I removed the icon property

Reply:
Since it doesn't look too pretty w/o having icons for the network and the token at the same time (see the first screenshot), I used `bitcoinTestnet` for the BTC Regtest token (see the second screenshot). Please let me know if you have other ideas here. 

<img width="220" alt="Screenshot 2024-09-09 at 10 55 43" src="https://github.com/user-attachments/assets/780323d5-b081-4039-a260-08a402c30786">

<img width="222" alt="Screenshot 2024-09-09 at 11 00 19" src="https://github.com/user-attachments/assets/5d65bb0a-7d7c-47fd-8f1b-14c20c7f77df">

Reply:
I don't have a strong opinion on one or the other. @peterpeterparker your call.

Reply:
Sounds good.